ann_benchmarks/__init__.py

```diff
@@ -213,18 +213,28 @@ def __init__(self, metric, P):
         self._metric = metric
 
     def fit(self, X):
+        os.environ['OMP_THREAD_LIMIT'] = '40'
```

Comment:
not sure if i love this, but i guess it's fine for now

i'm trying to make up my mind whether we should limit ann-benchmarks to 1 core or not. one idea i've been thinking about is let's not impose any limitations but run `query` using a thread pool – only downside is if thrashing kills the L2/L3 cache